wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+
+## Maximiser la qualité et diminuer le délai
+
+### 1. Utilisez une interface audio∕un microphone externe, pas votre carte son interne
+
+Sauf dans le cas où vous êtes un musicien ayant un microphone (USB), il est fortement recommandé d'utiliser une carte son : un appareil sur lequel vous pouvez brancher votre instrument. Celle-çi se connectera à votre PC, généralement par USB. Visitez [la section matériel](Hardware-Setup).
+
+### 2. Réduisez votre trafic Internet et fermez les programmes de votre PC
+
+Assurez-vous que Jamulus ait le maximum de ressources sur votre machine ainsi que via votre connexion réseau quand vous l'utilisez.
+
+### 3. Connectez-vous aux serveurs qui ont le meilleur temps de réponse
+
+Les serveurs qui s'affichent dans la liste avec un temps de 15ms ou moins optimiserons votre expérience (même s'il y a d'autres éléments qui influent sur les performances). Avec des temps de réponse plus longs, c'est à vos risques et périls.
+
+Maintenant que vous avez les bases, c'est parti :
+
+<div class="fx-row fx-row-start-xs button-container">
+  <a href="Installation-for-Windows" class="button fx-col-100-xs">Installer sur Windows</a>
+  <a href="Installation-for-Macintosh" class="button fx-col-100-xs">Installer sur macOS</a>
+  <a href="Installation-for-Linux" class="button fx-col-100-xs">Installer sur une distribution GNU/Linux</a>
+</div>
+
+Découvrez aussi : [Jamulus OS !](https://sourceforge.net/projects/jamulus-os/files/JamulusOS/) Exécutez Jamulus sur n'importe quel ordinateur avec une clé USB (uniquement pour les systèmes en 64 bits). Vérifiez la compatibilité de votre matériel et profitez-en !
+
+## Tout est bon ?
+
+Si vous voulez configurer Jamulus, visitez [la section matériel](Hardware-Setup).
+
+Pour de plus amples informations à propos de l'utilisation de Jamulus, regardez [le mode d'emploi](https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md).
+
+## D'autres guides
+* [Cet excellent guide (en)](https://www.facebook.com/notes/jamulus-online-musicianssingers-jamming/idiots-guide-to-jamulus-app/510044532903831/){: target="_blank" rel="noopener noreferrer"} par [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a){: target="_blank" rel="noopener noreferrer"} (_Facebook_) pourrait aussi vous aider.
+
+## Un ou des problème(s) ? Ça ne fonctionne pas longtemps ?
+
+**ÉCOUTEZ. UNIQUEMENT. LE. SIGNAL. DE. VOTRE. SERVEUR !**
+
+Ou dit autrement : n'écoutez **pas** votre signal directement.
+
+C'est évidemment compliqué pour les chanteurs et pour certains instruments, mais même si votre connexion est excellente, si les chanteurs/musiciens n'écoutent pas leur propre son provenant du serveur, votre synchronisation en pâtira. (Attention : cela n'a rien à voir avec les commandes « mutes » de Jamulus)
```

Comment:
« Ça peut évidement être compliqué pour les chanteurs ou les chanteuses […] si les chanteurs ou les musiciens n'écoutent pas […] »

Reply:
* "pour certains instruments" -> "pour certains instrumentistes"
* "chanteurs/musiciens" -> "joueurs"
* "leur propre son provenant du serveur" -> "**leur propre son provenant du serveur**" (in bold)
* "en pâtira" -> "**en pâtira**" (in bold too)
* "(Attention : cela n'a rien à voir avec les commandes « mutes » de Jamulus)" -> "_Pour éviter tout doute : cela n'a rien à voir avec les contrôles "muets" de Jamulus._" (rewording suggestion + do not forget the italic)

Reply:
> "chanteurs/musiciens" -> "joueurs" : I don't like to say "joueur" only but "joueur d'un instrument" it's fine but long…

Reply:
@Saimyx say
> I don't like to say "joueur" only but "joueur d'un instrument" it's fine but long…

Let's take another look at the original sentence in English:
> This is obviously hard for singers and some acoustic musicians, but even if your connection is good, when players aren’t listening to their own signal coming back from the server, your synchronisation will suffer. (For the avoidance of doubt: this has nothing to do with the “mute” controls in Jamulus.)

The literal translation (word for word) is :
« C'est évidemment difficile pour les chanteurs et certains musiciens acoustiques, mais même si votre connexion est bonne, lorsque les joueurs n'écoutent pas leur propre signal revenant du serveur, votre synchronisation en souffre. (Pour éviter tout doute : cela n'a rien à voir avec les commandes "mute" de Jamulus). »

Using « mute » instead of « silencer » or « muet » is usual because of the UI of the sound devices.
In the French UI of Jamulus « muet » est utilisé, so we must use « muet ».

I don't like using « joueurs » for « players » in the context. In the context we talk about musicians, so « musicien(s) » sounds good.

So : 
* _This is obviously hard for singers and some acoustic musicians_ ->  
  * C'est évidemment difficile pour les chanteurs et certains musiciens acoustiques,  
or
  * C'est évidemment difficile pour les chanteurs et certains instruments acoustiques,

If we take care about womens we should use when we can « chanteurs ou chanteuses » (or « musiciens ou musiciennes ») if we don't want to use inclusive writing.

* _when players aren’t listening_ -> lorsque les musiciens n'écoutent pas
because we use present above we can't use futur after in the same sentence, so from your initial proposal I suggest: 

-------
Cela peut évidemment être difficile pour les chanteurs ou chanteuses et certains instruments acoustiques, mais même si votre connexion est excellente, lorsque les musiciens ou musiciennes n'écoutent pas leur propre signal revenant du serveur, la synchronisation en pâtit.  
(_Pour éviter toute confusion : cela n'a rien à voir avec les commandes « muet » de Jamulus_). 

-------
_excellente_ instead of _bonne_ gives more emphasis to the warning, IMHO.

Take care about the quotation marks : « » that's we must use (with no breaking space after the opening and before the closing, according the typographical rules)
The same before ? or !, with the markdown we can use the `&#160`or `U+00A0` (ALT+MAJ+SPACE on MacOS), in HTML `&nbsp;`.
We can't only for title when we have internal links, that doesn't works with these in the text. (I've make a lot of tests!)